examples/chip-tool/commands/common/Command.cpp

```diff
@@ -877,9 +877,8 @@ void ResetOptionalArg(const Argument & arg)
 
 void Command::ResetArguments()
 {
-    for (size_t i = 0; i < mArgs.size(); i++)
+    for (auto arg : mArgs)
```

Comment:
Can this be `const auto & arg`?

Reply:
Will go through renames manually and update the names. Will update this to const as well.